test/relations.test.js

```diff
@@ -34,9 +34,16 @@ describe('relations', function() {
     db = getSchema();
   });
 
+  after(function(done) {
+     // disconnect from this db to avoid too many connection error
+     // due to multiple instance of connection pool
+    db.disconnect();
+    done();
+  });
+
   describe('hasMany', function() {
     before(function(done) {
-      db = getSchema();
+      // db = getSchema();
```

Comment:
@ebarault Cheers that only cloudant fails now! While I am confused here, why do we use previous datasource instead of getting a new datasource instance?

I understand `db.disconnect()` is necessary, but using one datasource instance in all tests will result in old models will always be attached to that datasource. Which means when reaching the last describe suite, the db will have tens of models attached to it and `automigrate` all of them :p